src/pages/characters/[[...params]].js

```diff
@@ -0,0 +1,77 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { Typography } from "@material-ui/core"
+import { useSelector } from "react-redux"
+import { useRouter } from "next/router"
+import { find } from "lodash"
+
+import CustomTable from "@/components/CustomTable"
+import AddNewButton from "@/components/AddNewButton"
+import RightSideDrawer from "@/components/RightSideDrawer"
+import CharacterForm from "@/components/CharacterForm"
+import { characters } from "@/selectors/characterSelectors"
+import { formEnum } from "@/enums/formEnum"
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: "grid",
+    placeItems: "center",
+    gap: "10px",
+    padding: "25px",
+  },
+  button: {
+    placeSelf: "end",
+  },
+}))
+
+export default function Home() {
+  const classes = useStyles()
+
+  const data = useSelector(characters)
```

Comment:
When using selectors, it's best practice to name the variable after the piece of data you are selecting, so this should be `const characters = useSelector(characterSelectors.characters)`. You should also be importing all to avoid the name clash